pom.xml

```diff
@@ -433,6 +428,21 @@
             <artifactId>lambdaj</artifactId>
             <version>2.3.3</version>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>${logback.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>${logback.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.janino</groupId>
```

Comment:
Thank you for your submission! Why did you add org.codehaus.janino here?

Reply:
The latest version of logback requires org.codehaus.janino to evaulate conditional logic per https://stackoverflow.com/questions/11017148/how-does-the-condition-functionality-work-in-logback. In this case the logback config doesn't use conditional logic so it is not needed. I will remove it.

Reply:
Hmm actually there are conditionals like `<if condition='isDefined("color")'>` not sure how they work without org.codehaus.janino

Reply:
@spacether 
I believe I've preemptively answered all your questions in the [summary](https://github.com/openapi-json-schema-tools/openapi-json-schema-generator/pull/317#issue-2023653355) at the top of my PR.

Glad you eventually figured out why the addition of`janino` was necessary.